lib/stripe_mock/data.rb

```diff
@@ -374,15 +374,20 @@ def self.mock_subscription(params={})
           data: [{
             id: 'si_1AwFf62eZvKYlo2C9u6Dhf9',
             created: 1504035973,
-            metadata: {},
+            metadata: {
+              # Added for avoiding nil exception
+              company_deleted: false
```

Comment:
This is meta-data that we write when deleting a company; this does _NOT_ belong in the gem. We need to remove this and adjust our tests/designs accordingly.